### ISSUE TITLE
vra_project -> zone_assignments ignore multiple values

### ISSUE THREAD

**Issue by MichalZezula** [vmware/terraform-provider-vra#305]
**vRA version**
vRealize Automation 8.2.0.13070 (17151459)
**Terraform version**
Terraform v0.14.5
**terraform-provider-vra plugin version**
provider registry.terraform.io/vmware/vra v0.3.4

**Describe the bug**
zone_assignments in vra_project data resource do not display more than one zone assignment

**To Reproduce**
- Fetch existing vra_project with multiple zone assignments
- Check tfstate if it contains all zone_assignments
```
data "vra_project" "X"{
  name = "X"
}
```

**Expected behavior**
I expect to see all zone assignments as visible in web ui

**Screenshots**
![image](https://user-images.githubusercontent.com/77154285/107525262-f5621d80-6bb6-11eb-863c-504b1ca1881b.png)
![image](https://user-images.githubusercontent.com/77154285/107525375-17f43680-6bb7-11eb-9ec4-674198f26ff6.png)

**Desktop (please complete the following information):**
 - OS: Win 10 PRO

**Comment by Prativa20** [comment]
Hi @MichalZezula I am not able to reproduce it with the latest version. Can you please verify it once again?
![Screen Shot 2021-02-11 at 11 11 50 AM](https://user-images.githubusercontent.com/10183678/107686762-79211600-6c5a-11eb-8c0b-fb94942f37a6.png)

**Comment by MichalZezula** [comment]
Hello,
Just to be sure I've updated Terraform to the very latest version -> 14.6 .
I've tried it again but seeing the same result.
I've also tried to define project in terraform with multiple zone assignments and this seems to work perfectly fine. But when I try to fetch these informations using data resource I'm seeing only one of the assignments.


[vra_project_redacted.tfstate.txt](https://github.com/vmware/terraform-provider-vra/files/5970352/vra_project_redacted.tfstate.txt)

**Comment by Prativa20** [comment]
@MichalZezula I could reproduce this issue. This looks like a terraform issue and not a provider issue. In the schema, zone_assignments is a TypeSet that might be causing all the zones to be not saved in the state file. I changed the data type to TypeList and it works fine. I have opened a terraform bug for this https://github.com/hashicorp/terraform/issues/28169

However, I will do some more testing to see nothing breaks before changing it to TypeList.

Thanks,
Prativa
